app.json

```diff
@@ -11,15 +11,19 @@
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },
-    "assetBundlePatterns": ["**/*"],
+    "assetBundlePatterns": [
+      "**/*"
+    ],
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.lbare.uhub"
```

Comment:
you need to change the bundle identifier for iOS to the incoming change

Reply:
Alright, merge resolved. Pretty much the PR is just a readme update now lol